src/main/java/com/google/sps/classes/Utility.java

```diff
@@ -207,6 +212,50 @@ public static Question buildQuestion(ResultSet queryResult) {
     return question;
   }
 
+  /**
+   * Returns true if mentor review is approved or rejected.
+   */
+  public static int isReviewed(boolean isApprovalType, int mentorId) {
+    String reviewType = "";
+    if (isApprovalType) {
```

Comment:
to make it more readable, having a String approvalType might be better

Reply:
you could even make ```reviewType=approvalType``` and skip the if

Reply:
Oh, wow, great idea! Done.